source/NVDAObjects/window/winword.py

```diff
@@ -1331,6 +1331,8 @@ def script_lineSpacing(self,gesture):
 		elif val == wdLineSpaceDouble:
 			# Translators: a message when switching to double space  in Microsoft word
 			ui.message(_("Double Spaced"))
+	# Translators: Input help mode text for a command available in Microsoft word.
+	script_reportCurrentComment.__doc__=_("changes the line spacing of the currently selected text.")

```

Comment:
To remove a string, should I or should I not include a docstring?


On 5/25/2016 5:07 PM, Michael Curran wrote:
>
> In source/NVDAObjects/window/winword.py 
> <https://github.com/nvaccess/nvda/pull/5995#discussion_r64668568>:
>
> > @@ -1331,6 +1331,8 @@ def script_lineSpacing(self,gesture):
> >  		elif val == wdLineSpaceDouble:
> >  			# Translators: a message when switching to double space  in Microsoft word
> >  			ui.message(_("Double Spaced"))
> > +	# Translators: Input help mode text for a command available in Microsoft word.
> > +	script_reportCurrentComment.__doc__=_("changes the line spacing of the currently selected text.")
>
> You've put the docstring on the wrong script ;) Also, as this script 
> only makes an existing application command more accessible, the 
> docstring is not as necessary anyway.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub 
> <https://github.com/nvaccess/nvda/pull/5995/files/cec39f17432319f8345a50524a883ca9f63e7406..dae5223186417e7ee6d717a19c935d835b7445fe#r64668568>
>

-- 
------------------------------------------------------------------------


    Derek Riemer

  * Department of computer science, third year undergraduate student.
  * Proud user of the NVDA screen reader.
  * Open source enthusiast.
  * Member of Bridge Cu
  * Avid skiier.

Websites:
Honors portfolio <http://derekriemer.com>
Awesome little hand built weather app! 
<http://django.derekriemer.com/weather/>

email me at derek.riemer@colorado.edu <mailto:derek.riemer@colorado.edu>
Phone: (303) 906-2194